Sources/Sentry/Public/SentryDefines.h

```diff
@@ -142,3 +142,84 @@ typedef NS_ENUM(NSInteger, SentryTransactionNameSource) {
     kSentryTransactionNameSourceComponent,
     kSentryTransactionNameSourceTask
 };
+
+/**
```

Comment:
Missing the type `none` which is valid and part of the spec.
This is necessary because of https://github.com/getsentry/sentry-cocoa/pull/2268/files#r989907253